hw/pci-bridge/cxl_socket_transport.c

```diff
@@ -20,6 +20,13 @@
 #define MAX_PAYLOAD_SIZE 512
 #define MAX_DURATION     5
 
+// For mreq_header_t endianness compatibility
+#define EXTRACT_UPPER_56(address) ((address >> 6))
```

Comment:
Could you use uint64_t extract64(uint64_t value, int start, int length) instead?

https://www.qemu.org/docs/master/devel/bitops.html#c.extract64